### PR TITLE
feat: add grids

### PR DESCRIPTION
**Pull request by devdeadviz**
# Change Logs
## Screenshots
Screenshot (141)
## Deploy Preview Link:
https://deploy-preview-9--arcui.netlify.app/pages/components/grid/grid.html

### CODE REVIEW

**Review by xii-vi — APPROVED** [review]
Grids are fine to me.
you can merge them

**Review by nyyshaaa — COMMENTED** [review]
looks good to merge!

File: pages/components/grid/grid.html [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+              <a
+                target="_blank"
+                class="social-icons-wrapper"
+                href="https://twitter.com/devdeadviz"
+              >
+                <i class="fab fa-twitter"></i>
+              </a>
+            </li>
+            <li>
+              <a
```
Comment (ckahub) [inline pages/components/grid/grid.html:31]: here extra space is used in anchor tag , can be avoided

File: pages/components/grid/grid.html [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+                target="_blank"
+                class="social-icons-wrapper"
+                href="https://www.linkedin.com/in/devdeadviz/"
+              >
+                <i class="fab fa-linkedin"></i>
+              </a>
+            </li>
+            <li>
+              <a
+                target="_blank"
```
Comment (ckahub) [inline pages/components/grid/grid.html:41]: same here

**Review by ckahub — APPROVED** [review]
looks good to me

File: pages/components/grid/grid.html (hunk 1 above)
Comment (devdeadviz) [reply pages/components/grid/grid.html:31]: It's not the extra space.

**Pull request merged**